.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         runs-on: [ubuntu-latest, macos-latest, windows-latest]
 
         include:
-          - python-version: pypy-3.8
+          - python-version: pypy3.8
```

Comment:
this change is needed as `hatch` doesn't seem to recognize `pypy-3.8` as a python version but was happy with `pyp3.8` which is supported by GitHub Actions (see actions/setup-python#349).